{"event": "verbose", "uuid": "854aa5f8-3a3d-44b2-8c7d-30399de69ff5", "counter": 13, "stdout": "\u001b[0;31mThe error appears to be in '/home/u/my_project/ping/test.yml': line 9, column 9, but may\u001b[0m", "start_line": 12, "end_line": 13, "runner_ident": "aae58ea8-7b28-47a4-a8c1-018cc295dc5a", "created": "2024-05-21T08:53:56.823532"}